### PR TITLE
display table records in an html table and make it look like the mocks.

### PR DESCRIPTION
**Pull request by davidsbailey**
Display table records in an html table and make it look like the mocks. Provide 'records' child of table data directly via redux. does not yet include add/edit/save/delete UI actions.

screencap:
table-view

### CODE REVIEW

File: apps/src/applab/dataBrowser/AddTableRow.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          this.props.columnNames.map(columnName => (
+            <td key={columnName} style={dataStyles.cell}>
+              { (columnName !== 'id') && <input style={dataStyles.input}/> }
+            </td>
+          ))
+        }
+        <td style={dataStyles.cell}>
+          <button
+            className="btn btn-primary"
+            style={dataStyles.button}
```
Comment (Bjvanminnen) [inline apps/src/applab/dataBrowser/AddTableRow.jsx]: style guide prefers four spaces of indent for attributes (throughout PR)

**Comment by Bjvanminnen** [comment]
one nit, otherwise lgtm :)

File: apps/src/applab/dataBrowser/AddTableRow.jsx (hunk 1 above)
Comment (davidsbailey) [inline apps/src/applab/dataBrowser/AddTableRow.jsx]: I haven't been able to make RubyMine do this for me, but it looks like we might be able to catch this in precommit hooks via https://github.com/yannickcr/eslint-plugin-react

Comment (davidsbailey) [inline apps/src/applab/dataBrowser/AddTableRow.jsx]: or maybe we can just use `eslint@es6jsx`: http://eslint.org/blog/2014/11/es6-jsx-support

**Pull request merged**